src/minimax_server/router.clj

```diff
@@ -0,0 +1,18 @@
+(ns minimax-server.router
+  (:import [java.util.function Function]
+           [scarvill.httpserver.response ResponseBuilder])
+  (:require [clojure.string :as str]))
+
+(defn response-with-body [body]
+  (.build (.setBody (new ResponseBuilder) (byte-array (map byte (str body))))))
+
+(defn get-current-player [request]
+  (.getParameterValue request "current_player"))
+
+(defn get-board [request]
+  (str/split (.getParameterValue request "board") #","))
+
+(defn minimax-router [get-move]
+  (proxy [Function] []
+    (apply [request]
+           (response-with-body (get-move (get-current-player request) (get-board request))))))
```

Comment:
A couple of questions:

1. What would you expect to happen if you were a client of this service and you sent a bad response?
2. Is the specified behavior currently tested?
3. What is the motivation for making the suggested refactoring a story? I'm not sure I see the business value in adding an additional layer of abstraction.